app/middlewares/db_middleware.py

```diff
@@ -18,17 +16,25 @@
 
 
 class DBMiddleware(BaseMiddleware):
-    def __init__(self, tg_client_config: TgClientConfig):
+    def __init__(self):
         super(DBMiddleware, self).__init__()
         self.lock_factory = LockFactory()
-        self.tg_client_config = tg_client_config
+
+    async def __call__(self, handler: Callable[[TelegramObject, Dict[str, Any]], Awaitable[Any]], event: TelegramObject,
+                       data: Dict[str, Any]) -> Any:
+        chat: types.Chat = data.get("event_chat", None)
+        user: types.User = data.get("event_from_user", None)
+        if isinstance(event, types.Message) and event.sender_chat:
+            raise CancelHandler
+        await self.setup_chat(data, user, chat)
+        return await handler(event, data)
 
     async def setup_chat(self, data: dict, user: types.User, chat: Optional[types.Chat] = None):
         try:
-            async with self.lock_factory.get_lock(f"{user.id}"):
+            async with self.lock_factory.get_lock(str(user.id)):
                 user = await User.get_or_create_from_tg_user(user)
             if chat and chat.type != 'private':
-                async with self.lock_factory.get_lock(f"{chat.id}"):
+                async with self.lock_factory.get_lock(str(chat.id)):
```

Comment:
Does it really need cast to `str`?

Reply:
remove this cast, probably it will be ok